docs/database/README.md

```diff
@@ -107,7 +114,8 @@ mysql> CREATE USER 'submarine_test'@'%' IDENTIFIED BY 'password_test';
 mysql> GRANT ALL PRIVILEGES ON * . * TO 'submarine_test'@'%';
 mysql> CREATE DATABASE `submarineDB_test` CHARACTER SET utf8 COLLATE utf8_general_ci;
 mysql> use `submarineDB_test`;
-mysql> source ./docs/database/submarine.sql;
+mysql> source /submarine.sql;
+mysql> source /submarine-data.sql;
```

Comment:
Cannot execute `submarine-data.sql` in the `submarineDB_test` database,
Because the data in `submarineDB_test` is all generated by the test case.